v2/contrib/urfave/negroni/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/DataDog/dd-trace-go/v2/contrib/urfave/negroni
 
 go 1.19
 
+replace github.com/DataDog/dd-trace-go/v2 => ../../../../
+
```

Comment:
@knusbaum PR looks good in general. About this replace directives, if you merge v2-dev, all contribs now have them too.

Reply:
Thanks, that's very good to know.